src/frontend/src/common/components/library/data_subview/components/EditSamplesConfirmationModal/index.tsx

```diff
@@ -82,8 +84,19 @@ const EditSamplesConfirmationModal = ({
   };
 
   const handleClose = function () {
+    setLoseProgressModalOpen(true);
```

Comment:
should onClose also happen here? If not, maybe we want to rename the prop to something more semantic, to indicate it doesn't always happen when the modal closes.